generators/app/index.js

```diff
@@ -103,25 +125,38 @@ module.exports = class extends Generator {
     }
   }
 
+  _initModuleName() {
```

Comment:
Let's find a better place for this logic. I don't think the side-effets it has because it's making it hard to know exactly when these values end up being created. In the future, I fear this will be hard to debug.

Reply:
I have removed the side effects from the method and moved them to `_askForModuleName`, wherein `this.props.name` was already being set. `_askForModuleName` only has side effects, so this should be fine; otherwise, I’d suggest moving the setting of the props out of the `_ask*`-methods and into the `prompting`-method.